### DISCUSSION TITLE
Why is the embedded Youtube video not responsive?

### DISCUSSION THREAD

**Discussion by pawell0** [datenstrom/community#643] [Ask a question] [answered]
I have a little problem or this is how the Youtube plugin works. The embedded video is not resposive. I'm using the "Paris" theme. What can I do to have the published videos responsive on my website? I'm using the newest Yellow CMS with the newest Youtube plugin and same with the "Paris" theme. All without any modifications. I noticed that only the video without given the width and height parameters is responsive. What should I do to fix this?

**Comment by nzyeehaa** [comment, marked as answer]
sounds like you may need a web designer,

you said the video is not responsive BUT you have already discovered that with no fixed width - it is responsive :-)
sounds great to me...

if you are using the youtube extension - a fixed width can be set - or leave it out [- see extensions/youtube](https://github.com/datenstrom/yellow-extensions/tree/master/source/youtube)

or start adjusting the css 
( ! warning ! - this can be like squeezing a balloon - and requires extensive cross browser testing)

.flexible iframe { position:absolute; top:0; left:0; width:100%; height:100%; }  // has worked in the past

**Reply by pawell0** [reply]
Hello, [nzyeehaa](https://github.com/nzyeehaa). Thanks for answering my post. My thought was: why the video is not responsive with fixed width, when entire website is responsive. Maybe my way of thinking is not correct. ;-) Thank you for the piece of code, I will inspect it when it will be necessary. Greetings.

**Reply by nzyeehaa** [reply]
your welcome,
(sorry link above was broken - fixed now) - it shows `Width = video width, pixel or **percent**`

try [youtube fhs55HEl-Gc 80%]

otherwise - you will be "going down the rabbit hole" chasing CSS code...

when you tell a video to be 640px wide - it stays that way
when you tell a video to be 50% wide - it can be 50% of the box it is in (on most browsers) and you are closer to being responsive - but may be too small on a phone

The yellow templates are lightweight(size) and they cater fairly well for general use.
